injectionapi/socketlabsclient.go

```diff
@@ -6,10 +6,11 @@ import (
 	"net/http"
 	"net/url"
 	"runtime"
+	"time"
 
-	"github.com/socketlabs/socketlabs-go/injectionapi/core"
-	"github.com/socketlabs/socketlabs-go/injectionapi/core/serialization"
-	"github.com/socketlabs/socketlabs-go/injectionapi/message"
+	"github.com/PraneethChandraThota/socketlabs-go/injectionapi/core"
```

Comment:
after your tests are complete please return the imports back to the main repo

Reply:
Changed back to the main repo